### PR TITLE
Install centos iso

### PR DESCRIPTION
**Pull request by heckj**
# CentOS installation from DVD

updating README instructions to working through the process of downloading a CentOS ISO and how to run a workflow that installs that OS from the provided ISO.

During this setup, I opened #35 and #36 with issues related to this process.

### CODE REVIEW

**Comment by yyscamper** [comment]
Hi @heckj: Have you ever successfully install CentOS 7.0 via your steps? Actually, I found the ISO file is not workable, if you carefully check this folder (http://mirror.umd.edu/centos/7/os/x86_64/repodata/) with the same folder that extracted from ISO, you will see all files in the former folder have extension, but ISO doesn't.  But if miss the extension, the installation will fail.

**Comment by heckj** [comment]
Evening @yyscamper, yes - I did get it fully operational. This PR and the instructions within the README work from end to end, as an example of how to use an ISO to set up a repository and then install using our existing workflow and templates (with a needed tweak to accomodate a slightly different `repo` option for the layout and naming that is incurred from unpacking the ISO locally)

**Comment by heckj** [comment]
@benbp @yyscamper @anhou @jlongever @brianparry looking for reviews please.

File: example/README.md [hunk 1]
```diff
@@ -161,38 +161,61 @@ for rebooting the `pxe-1` virtual machine.
...
+the KVM packages after the default installation.
+
+**NOTE** because this demonstration setup uses Virtualbox, there is no out
+of band management to trigger the machine to reboot. Once the workflow
+has been activated, the VM will need to be rebooted manually for the
+workflow to operate.
+
+## HACKING THESE SCRIPTS
+
+The vagrat image is built using [packer](https://packer.io/) with the
```
Comment (benbp) [inline example/README.md]: "vagrat"

File: example/README.md [hunk 2]
```diff
@@ -161,38 +161,61 @@ for rebooting the `pxe-1` virtual machine.
...
-    sudo python ~/src/on-http/data/templates/setup_iso.py /vagrant/Cent*.iso \
-    /var/mirrors --link=/home/vagrant/src
+    cd /tmp
+    wget http://mirror.umd.edu/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1503-01.iso
+    # 4GB
+    sudo python ~/src/on-http/data/templates/setup_iso.py /tmp/CentOS-7-x86_64*.iso /var/mirrors --link=/home/vagrant/src
 
-## HACKING THESE SCRIPTS
+The CentOS installer wants a bit more memory easily available for the
+installation than we default our test VM towards, so I recommend updating
```
Comment (benbp) [inline example/README.md]: We instead of "I" here?

**Comment by benbp** [comment]
:+1:

File: example/samples/centos_iso_boot.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Graph.InstallCentOS",
```
Comment (rolandpoulter) [inline example/samples/centos_iso_boot.json]: Just curious if this is a sample of a workflow or task?

**Comment by DavidjohnBlodgett** [comment]
:+1:

File: example/samples/centos_iso_boot.json (hunk 3 above)
Comment (heckj) [inline example/samples/centos_iso_boot.json]: Sample workflow invocation that works with unpacking the CentOS DVD ISO

**Comment by anhou** [comment]
:+1:

**Pull request merged**